src/hooks/useRequest.tsx

```diff
@@ -103,8 +104,14 @@ export const useRequest = <
 			return;
 		}
 		setData(undefined);
+	}, [...(options?.resetDataDependencies ?? deps), polling]);
```

Comment:
Hey @thgreasi sorry to bother you again, do you agree on adding this option to the useRequest hook ? this will allow us to exclude pagination properties from resetting data, and will avoid having the table resizing every pageChange (switching from empty to full)